site/content/blog/release-0.30.0.md

```diff
@@ -0,0 +1,134 @@
+
+- The `util` module is all but gone, try just removing `util::` at first, most modules are
+  re-exported at the crate root.
+
+- Cryptography related stuff is now in the `crypto` module, all submodules are re-exported so
+  you can import them directly (e.g. `use bitcoin::ecdsa`). Modules include: (`ecdsa`, `taproot`,
+  `sighash`, `key`)
+
+- Some hash wrapper types are now to be found in the module that they are used in e.g.,
+  `TapLeafHash` is in the `taproot` module. Others are still in `hash_types` but the re-exports now
+  conform to the aim stated above so you might need to add `hash_types::` to your paths for the more
+  esoteric hash types.
+
+## sighash
+
+We moved around and renamed a bunch of types to do with sighashes. In the `sighash` module, along
+with the `SighashCache` we now have the hopefully clearly named:
+
+- `LegacySighash`
+- `SegwitV0Sighash`
+- `TapSighash`
+- `EcdsaSighashType`
+- `TapSighashType`
+
+Signatures are now in their respective modules (`ecdsa` for legacy and segwit v0):
+
+- `taproot::Signature`
+- `ecdsa::Signature`
+
+## Lock types
+
+There are now two lock times, one for absolute locks (CLTV) and one for relative locks (CSV). We
+export the `absolute` and `relative` modules at the crate root so you can either import them from
+their or `use bitcoin::locktime::{absolute, relative};` if that's clearer. We expect locks to be
+used as `absolute::LockTime`.
+
+## Script changes
+
+We added a `ScriptBuf` alongside `Script`, these two now represent owned and borrowed types the same
+way that `PathBuf` and `Path` do (or `String` and `str`) i.e., we pass `&Script` if we only need to
+borrow.
+
+This change is quite invasive but should hopefully help to improve APIs here and in crates depending
+on us.
+
+## Address changes
+
+Bitcoin addresses for different networks are different, up until this release when parsing an
+address from a string there were no checks that the address format matched up to the expected
+network (e.g. "bc1" prefix for Bitcoin mainnet segwit addresses).
+
+Now `Address<V>` includes a generic that is used as a marker for whether the address has been
+checked as valid for a particular network, we have `Address<NetworkChecked>` and
+`Address<NetworkUnchecked>`, defaulting to `NetworkChecked`. Because of the default some uses will
+just keep working but you should be aware that `Address` now means `Address<NetworkChecked>`. The
+string parsing functions return types as expected. See the docs on `Address` for more information.
+
+## Newtypes
+
+This is a non-exhaustive list of newtypes added this release:
+
+- `absolute::LockTime`
+- `absolute::Height`
+- `absolute::Time`
+- `ecdsa::SerializedSignature`
+- `ScriptBuf`
+- `Target`
+- `CompactTarget`
+- `Work`
+
+### Removed types
+
+- `PackedLockTime`
+
+This type was intended as an optimization of the absolute locktime using a `u32`, this turned out to
+be note such a great idea. Please not `absolute::LockTime` does not implement `Ord`.
```

Comment:
```suggestion
be not such a great idea. Please note `absolute::LockTime` does not implement `Ord`.
```

Reply:
Thanks for taking the time to read this man, it was a serendipity to see your review!

Reply:
Happy to help :)